functions/src/generate-sessions-speakers-schedule.js

```diff
@@ -0,0 +1,90 @@
+import * as functions from 'firebase-functions';
+import { firestore } from 'firebase-admin';
+import mapSessionsSpeakersSchedule from './schedule-generator/speakers-sessions-schedule-map'
+import mapSessionsSpeakers from './schedule-generator/speakers-sessions-map'
+
+export const sessionsWrite = functions.firestore.document('sessions/{sessionId}').onWrite( async (change, context) => {
+    return generateAndSaveData();
+});
+
+export const scheduleWrite = functions.firestore.document('schedule/{scheduleId}').onWrite( async (change, context) => {
+    const scheduleConfig = functions.config().schedule;
+    const scheduleEnabled = scheduleConfig && scheduleConfig.enabled === 'true';
+    return scheduleEnabled && generateAndSaveData();
```

Comment:
we do this check in `generateAndSaveData`

Reply:
yep, we do. But assume schedule was changed but wasn't yet enabled. That means, in ```generateAndSaveData``` it will regenerate speakers and sessions again, but we don't want to.

Reply:
okay, make sense :)